src/main/java/com/aws/greengrass/clientdevices/auth/api/AuthorizeClientDeviceActionEvent.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+public class AuthorizeClientDeviceActionEvent implements DomainEvent {
+    @Getter
+    private AuthorizationStatus status;
+
+    public enum AuthorizationStatus {
+        SUCCESS,
+        FAIL
+    }
```

Comment:
Just got an idea from this.

If we added more details to this event then we could add a listener to this which logs authZ events. This would normally be extremely noisy most of the time. It would be nice to control this via CDA config instead of regular log levels (enabling nucleus debug logs == RIP)